### ISSUE TITLE
Simulation objects not renamed when importing from pkml

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/MoBi#653]
If I import a simulation from pkml and another simulation with the same name already exists in my project: I'm asked to rename the imported simulation.
However it renames only the simulation itself and not the containing obejcts.
E.g., my project has a simulation S1 and I import S1.pkml. During import I rename S1 to S2:
<img width="400" src="https://user-images.githubusercontent.com/25061876/140035638-7301523d-c7d2-4e1a-9d43-9ccfa68961df.png">

If i open S2 and select "Define settings and run": all outputs still have the old simulation name (S1):
<img width="500" src="https://user-images.githubusercontent.com/25061876/140035840-5a0b7c82-9b23-468a-958d-a9ebcc7415c6.png">

If after that I rename the imported simulation manually (e.g. from S2 to S3): simulation objects are properly renamed:
<img width="450" src="https://user-images.githubusercontent.com/25061876/140036046-dadc9122-729a-4371-89dd-cd8bed0207ff.png">

Seems that the renaming during import does not trigger the proper rename task.

Probably the reason for #651

**Comment by PavelBal** [comment]
Yes this is still an issue and even more of an issue now that we have obs data <-> sim results mapping in simulation:

![image](https://github.com/user-attachments/assets/6f813946-3a0f-41f6-a5b5-b719f7154eaf)

The simulation name is not updated in the "Simulation outputs" of the "Observed Data" tab.

It is also not possible to create the "Predicted vs observed" or the "Residuals vs time" plots as they still reference the old simulation:

![image](https://github.com/user-attachments/assets/201baa3c-7454-4ab0-bdf6-390427988926)

**Comment by PavelBal** [comment]
Seems to be fixed already.
